proselint/score.py

```diff
@@ -28,43 +32,44 @@ def score(check=None):
     """
     tp = 0
     fp = 0
-
-    parent_directory = os.path.dirname(proselint_path)
-    path_to_corpus = os.path.join(parent_directory, "corpora", "0.1.0")
+    # corpus was removed in https://github.com/amperser/proselint/pull/186
```

Comment:
I forgot the PR history had a copy of the old corpus lying around. This is useful, actually.

Reply:
we should assemble a new one. The previous Version was a webscrape. There are free sources available now, mostly for AI.

https://github.com/bakwc/JamSpell/blob/master/test_data/sherlockholmes.txt
-> a good start!

https://wortschatz.uni-leipzig.de/en/download/
-> much to process